src/test/java/api/loans/OverrideRenewByBarcodeTests.java

```diff
@@ -56,31 +56,6 @@
   private static final String ACTION_COMMENT_KEY = "actionComment";
   private static final String RENEWED_THROUGH_OVERRIDE = "renewedThroughOverride";
 
-  @Test
-  public void cannotOverrideRenewalWhenLoanPolicyDoesNotExist() {
```

Comment:
It is still possible for the rules to refer to a non-existent notice policy when an overridden renewal is performed. Why has this test been removed?

Reply:
OK, I returned it back and little bit updated

Reply:
Thanks